src/clj/test/cards/operations.clj

```diff
@@ -273,6 +273,40 @@
     (play-from-hand state :corp "Diversified Portfolio")
     (is (= 7 (:credit (get-corp))) "Ignored remote with ICE but no server contents")))
 
+(deftest enhanced-login-protocol
+  ;; Enhanced Login Protocol - tests interaction with run events, interaction with runs, and inability to run last click
+  (do-game
+    (new-game (default-corp [(qty "Enhanced Login Protocol" 1)])
+              (default-runner [(qty "Dirty Laundry" 1)]))
+    (play-from-hand state :corp "Enhanced Login Protocol")
+    (take-credits state :corp)
+    (is (= 4 (:click (get-runner))) "Runner starts turn with 4 clicks")
+    (play-from-hand state :runner "Dirty Laundry")
+    (prompt-choice :runner "Archives")
+    (run-successful state)
+    (is (= 3 (:click (get-runner))) "Run event only takes 1 click")
+    (run-on state :hq)
+    (run-jack-out state)
+    (is (= 1 (:click (get-runner))) "first non-event run takes 2 clicks")
```

Comment:
Better to check this happens immediately after the run begins, not after it ends, in case ice allows spending clicks.